services/blockchain-app-registry/shared/metadata.js

```diff
@@ -381,14 +384,21 @@ const resolveTokenMetaInfo = async (tokenInfoFromDB) => {
 const getSupportedTokensFromServiceURLs = async (serviceURLs) => {
 	for (let i = 0; i < serviceURLs.length; i++) {
 		const tokenSummaryEndpoint = `${serviceURLs[i].http}/api/v3/token/summary`;
-		// eslint-disable-next-line no-await-in-loop
-		const { data: response } = await HTTP.request(tokenSummaryEndpoint);
+		try {
+			// eslint-disable-next-line no-await-in-loop
+			const { data: response } = await HTTP.request(tokenSummaryEndpoint);
+
+			if (response.data && response.data.supportedTokens) {
```

Comment:
Do we need the log? if not, then would it be better to update the checks so that it does not generate exception in first place as exception handling a bit costlier operation?

```suggestion
			const response = await HTTP.request(tokenSummaryEndpoint);

			if (!response || !response.data || !response.data.data) continue;
			if (response.data.data.supportedTokens) {
```